src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -291,6 +171,7 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
                                       "Profile saved, but unexpected error registering user", e) }
   }
 
+  // TODO(dmohs): DRY
```

Comment:
if it is still valid then enter a JIRA and remove the TODO, it is useless to anyone but you - and my guess is that it will be useless to you too in about a month ;)